infra.bs

```diff
@@ -933,6 +933,8 @@ SPACE.
 
 <p class=note>"Whitespace" is a mass noun.
 
+<p class=note>[[xml11#NT-S|XML's definition of whitespace]] omits U+000C FF.
```

Comment:
WebKit has a "ASCII whitespace without FF" definition that is used for JSON/HTTP/XML. Perhaps we should add the same definition to Infra?

See also https://fetch.spec.whatwg.org/#http.

And yeah, it's not clear to me that 1.1 is actually supported. Chromium and WebKit seem rather lenient and allow `data:text/xml,<?xml version="1.2"?><test/>` which I don't think is permitted by the XML specifications, and Gecko rejects that and also rejects `1.1`.